learn/filtering_and_sorting/geosearch.mdx

```diff
@@ -138,18 +171,24 @@ Meilisearch will rebuild your index whenever you update `filterableAttributes`.
 
 ### Usage
 
-Use the [`filter` search parameter](/reference/api/search#filter) along with `_geoRadius` or `_geoBoundingBox`. These are special filter rules that ensure Meilisearch only returns results located within a specific geographic area.
+Use the [`filter` search parameter](/reference/api/search#filter) along with `_geoRadius` and `_geoBoundingBox`. These are special filter rules that ensure Meilisearch only returns results located within a specific geographic area. If you are using GeoJSON for your documents, you may also filter results with `_geoPolygon`.
 
 ### `_geoRadius`
 
 ```
-_geoRadius(lat, lng, distance_in_meters)
+_geoRadius(lat, lng, distance_in_meters, resolution)
```

Comment:
The resolution is optional here, I don't know if we have a way to indicate that 🤔 
In command line application, they would put like that: `[resolution]`
But since we're already using brackets below, I don't think it's a good idea